snapshots/storage/bolt.go

```diff
@@ -366,18 +366,18 @@ func CommitActive(ctx context.Context, key, name string, usage snapshots.Usage,
 			return err
 		}
 		if err := bkt.DeleteBucket([]byte(key)); err != nil {
-			return errors.Wrap(err, "failed to delete active")
+			return errors.Wrapf(err, "failed to delete active snapshot %q", key)
 		}
 		if si.Parent != "" {
 			spbkt := bkt.Bucket([]byte(si.Parent))
 			if spbkt == nil {
-				return errors.Wrap(errdefs.ErrNotFound, "missing parent")
+				return errors.Wrapf(errdefs.ErrNotFound, "missing parent snapshot of %q", key)
 			}
 			pid := readID(spbkt)
 
 			// Updates parent back link to use new key
 			if err := pbkt.Put(parentKey(pid, id), []byte(name)); err != nil {
-				return errors.Wrap(err, "failed to update parent link")
+				return errors.Wrapf(err, "failed to update parent link from %q to %q", key, name)
```

Comment:
Isn't the previous parent id `pid` not `key` ? So the message should be something like: `"failed to update parent link of %q to %q", key, name` ?